build.gradle.kts

```diff
@@ -49,3 +51,45 @@ extra["ROKT_TAG_ID"] = getProperty("ROKT_TAG_ID", "defaultTagId")
 extra["ROKT_PUB_ID"] = getProperty("ROKT_PUB_ID", "defaultPubId")
 extra["ROKT_SECRET"] = getProperty("ROKT_SECRET", "defaultSecretId")
 extra["ROKT_CLIENT_UNIQUE_ID"] = getProperty("ROKT_CLIENT_UNIQUE_ID", "defaultClientId")
+
+configure(subprojects) {
+    pluginManager.withPlugin("com.vanniktech.maven.publish") {
```

Comment:
One way to make this string typesafe is get the pluginId into a variable and use it
`val pluginId = libs.plugins.maven.publish.get().pluginId`
So that it will use the typesafe value defined in our toml file.

Reply:
Thanks @thomson-t that's a really good shout, added